A3A/addons/core/CfgFunctions.hpp

```diff
@@ -403,6 +403,7 @@ class CfgFunctions
             class LOG_Airdrop {};
             class LOG_Helicrash {};
             class missionRequest {};
+            class fn_effects {};
```

Comment:
file name is fn_effects, the config however does not need the fn_ part.
Also, rename fn_effects to fn_LOG_Crashsite_Effects
```suggestion
            class LOG_Crashsite_Effects {};
```

Reply:
shoud be called something else, or simply moved to
![image](https://github.com/SilenceIsFatto/A3-Antistasi-Ultimate/assets/29953387/75ac2ff2-234b-439b-888f-bbe9dbbcae71)

Reply:
that'll work too